rotkehlchen/chain/evm/constants.py

```diff
@@ -9,6 +9,7 @@
 ETH_SPECIAL_ADDRESS = string_to_evm_address('0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE')
 ZERO_32_BYTES_HEX = '0x' + '0' * 64
 GENESIS_HASH = deserialize_evm_tx_hash(ZERO_32_BYTES_HEX)  # hash for transactions in genesis block
+EVM_ADDRESS_REGEX = r'\b0x[a-fA-F0-9]{40}\b'
```

Comment:
Can you also add some unit tests for matching (and not matching) of this pattern?